notes/stories/film/film-tripod.st.txt

```diff
@@ -0,0 +1,150 @@
+Collection: TRIPOD films
+========================
+
+:: Title
+TRIPOD films
+
+:: Date
+1959-2020
+
+:: Description
+The 99 films in the TuRnIng POint Dataset (TRIPOD). 
+
+:: References
+https://github.com/ppapalampidi/TRIPOD
+
+:: Collections
+Collection: TRIPOD films
+
+:: Component Stories
+movie: Alien (1979)
+movie: Alien Nation (1988)
+movie: Bonnie and Clyde (1967)
+movie: Die Hard (1988)
+movie: I Am Legend (2007)
+movie: Jaws (1975)
+movie: Jurassic Park (1993)
+movie: Men in Black (1997)
+movie: Moon (2009)
+movie: Panic Room (2002)
+movie: Pretty Woman (1990)
+movie: Star Wars: Episode I - The Phantom Menace (1999)
+movie: Superman (1978)
+movie: Terminator Salvation (2009)
+movie: The Dark Knight (2008)
+movie: The Shining (1980)
+movie: The Time Machine (1960)
+movie: The Truman Show (1998)
+movie: Total Recall (1990)
+movie: V for Vendetta (2005)
+movie: Vertigo (1958)
+movie: X-Men (2000)
+
+
+movie: Pretty Woman (1990)
+==========================
+
+:: Title
+Pretty Woman
+
+:: Date
+1990-03-23
+
+:: Description
+Pretty Woman is a 1990 American romantic comedy film directed by Garry
+Marshall, from a screenplay by J. F. Lawton. The film stars Julia Roberts and
+Richard Gere, and features Héctor Elizondo, Ralph Bellamy, Laura San Giacomo,
+and Jason Alexander in supporting roles.
+
+The film's story centers on Hollywood prostitute Vivian Ward and wealthy
+businessman Edward Lewis. Vivian is hired to be Edward's escort for several
+business and social functions, and their relationship develops during her
+week-long stay with him.
+
+The film's title Pretty Woman is based on the 1964 song "Oh, Pretty Woman" by
+Roy Orbison.
+
+:: References
+https://en.wikipedia.org/wiki/Pretty_Woman
+
+:: Collections
+Collection: TRIPOD films
+
+:: Ratings
+4 <paul>
+
+:: Choice Themes
```

Comment:
I suppose the choicest theme of this cheesy story is the old "love of a woman transforms a man" crap. At any rate it's safe to knock romantic love up, imho.